packages/wrangler/src/dev-registry.ts

```diff
@@ -9,20 +10,29 @@ import type { Config } from "./config";
 import type { HttpTerminator } from "http-terminator";
 import type { Server } from "node:http";
 
-const DEV_REGISTRY_PORT = 6284;
+// Safety of `!`: `parseInt(undefined)` is NaN
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+let DEV_REGISTRY_PORT = parseInt(process.env.WRANGLER_WORKER_REGISTRY_PORT!);
```

Comment:
Do we document these anywhere? Worth adding to the list?

Reply:
Yeah, I mean, there are a bunch of them now, right? e.g. `WRANGLER_LOG_SANTIZE`, `WRANGLER_SEND_METRICS`, etc. etc. If you're not putting those anywhere (yet?), then definitely no rush on this one either.